TransformationSystem/Agent/TransformationCleaningAgent.py

```diff
@@ -1,10 +1,11 @@
-""" :mod: TransformationCleaningAgent
+"""TransformationCleaningAgent cleans up finalised transformations.
 
-    =================================
+.. literalinclude:: ../ConfigTemplate.cfg
```

Comment:
This results in:
https://diracsailer.readthedocs.io/en/fixdoc/CodeDocumentation/TransformationSystem/Agent/TransformationCleaningAgent.html#id1

I also though about using the dirac.cfg, but I am not sure this works when agent names are repeated, and the ConfigTemplate needs to be there in any case.

Reply:
Cool ! 

The only problem is if you have subsections.
But I suppose we could keep some discipline, for example putting special comments in the ConfigTemplate

```
##BEGIN TransformationCleaningAgent
TransformationCleaningAgent{
[...]
}
##END

Reply:
Nice! That also means I can drop the `:append: }` shenanigans and properly `:dedent:` :)